### ISSUE TITLE
Unexpected wrap caused by lost floating point precision 

### ISSUE THREAD

**Issue by nextend** [qq15725/modern-screenshot#104]
### Steps To Reproduce

[https://codesandbox.io/p/sandbox/modern-screenshot-forked-zdw9p9?file=%2Findex.html%3A11%2C9](https://codesandbox.io/p/sandbox/modern-screenshot-forked-zdw9p9?file=%2Findex.html%3A11%2C9
)

1. Open the given sandbox
2. A B C D should be in single line

Problem: The last character gets wrapped into new line on the screenshot.

The problem caused by that the window.getComputedStyle() floating precision limited to 4 decimals. 

The div element with font family declaration is 74.140625px (getBoundingClientRect()) in the original rendering. In the SVG created by the library it gets 74.1406px and that small missing area cause the wrap.

Both window.getComputedStyle(temp1).width  and window.getComputedStyle(temp1).inlineSize reports '74.1406px'

**Comment by Gamewise** [comment]
@nextend do you know if there is a way to fix this?

**Comment by t-rexwave** [comment]
Getting this issue as well. Any ideas for a fix or workaround?

**Comment by nextend** [comment]
I do not think that there will be workaround. We switched over server side rendering with headless Chrome.

**Comment by nextend** [comment]
@qq15725 This problem still happens with complex example with the new version too.

https://codesandbox.io/p/sandbox/modern-screenshot-forked-4fdxk6?file=%2Fpackage.json&workspaceId=192a8b3e-2754-418a-b6f2-6bb0195a7dc0

**Comment by qq15725** [comment]
> @qq15725 This problem still happens with complex example with the new version too.
> 
> https://codesandbox.io/p/sandbox/modern-screenshot-forked-4fdxk6?file=%2Findex.html%3A17%2C23&workspaceId=192a8b3e-2754-418a-b6f2-6bb0195a7dc0

The url no access permissions

**Comment by nextend** [comment]
I made it public.

**Comment by qq15725** [comment]
v4.5.1 looks normal


<img width="1077" alt="image" src="https://github.com/user-attachments/assets/ead4e97e-6542-4d5c-9274-6e7af4199511">

**Comment by nextend** [comment]
This is how it looks for me:
![image](https://github.com/user-attachments/assets/e08bf0c8-c023-4d9f-ba3f-7af0aac8bd42)

**Comment by dextrous-is-fun** [comment]
So excited for a fix for this if there is one!

**Comment by luruke** [comment]
I'm experiencing same issue on Android 10, chrome 130 :(

**Comment by luruke** [comment]
As temporary solution I ended up scaling down font size in copyCssStyles by 0.999

**Comment by pgsill** [comment]
@luruke 
> As temporary solution I ended up scaling down font size in copyCssStyles by 0.999

How exactly did you do this?

**Comment by Gamewise** [comment]
I saw this one got fixed, bu then the fix got reverted? Is that right?

**Comment by dextrous-is-fun** [comment]
Is there any update on this one? The unexpected line-wrap is causing issues for us :(

**Comment by BugCreators** [comment]
As a temporary workaround, you can try assigning a special class name to the target node. For example:

```js
<div class="font-size-fix"></div>
```

Then, for v4.6.5 and above:

```js
domToCanvas(dom, {
  onCloneEachNode: node=> {
    if (node.classList.contains('font-size-fix')) {
      node.style.fontSize = node.style.fontSize.replace(/(\d+(\.\d+)?(e[+-]?\d+)?)/g, (match, number) => {
       const parsedNumber = parseFloat(number);
    
       return isNaN(parsedNumber) ? match : (parsedNumber * 0.999).toString();
     });
    }
  }
})
```

For earlier versions:

```js
domToCanvas(dom, {
  onCloneNode: node=> {
    [...node.querySelectorAll('.font-size-fix')].forEach(item => {
      // xxx
    });
  }
})
```

**Comment by tyranasaurus** [comment]
Is it confirmed that this is fixed in v4.6.8? I found [ed8d711](https://github.com/qq15725/modern-screenshot/commit/ed8d711) which supposedly fixed this, but I'm having a similar behavior that I've found very difficult to debug in 4.6.8
